sessionize/lib/src/commonTest/kotlin/co/touchlab/sessionize/AppContextTest.kt

```diff
@@ -12,7 +9,7 @@ import kotlin.test.Test
 import kotlin.test.assertNotEquals
 import kotlin.test.assertTrue
 
-abstract class AppContextTests {
+abstract class StaticFileLoaderTest {
```

Comment:
Need to rename the File. These tests remain the same. We just don't have to deal with AppContext to perform them anymore